### PR TITLE
TASK-53952: Implement add new type request UI

### PR DESCRIPTION
**Pull request by hakermi**
Implement add new type request UI, component, forms, design etc..

### CODE REVIEW

File: processes-webapp/src/main/resources/locale/portlet/Processes_en.properties [hunk 1]
```diff
@@ -17,4 +17,36 @@ processes.myRequests.status.pendingLabel=Pending
 
 processes.myRequests.label.lastComment=Last comment
```
Comment (mkrout) [inline processes-webapp/src/main/resources/locale/portlet/Processes_en.properties]: Please rename requests to works

Comment (hakermi) [reply processes-webapp/src/main/resources/locale/portlet/Processes_en.properties]: done

File: processes-webapp/src/main/webapp/vue-app/processes/components/AddWorkFlowDrawer.vue [hunk 2]
```diff
@@ -0,0 +1,293 @@
...
+      <template slot="content">
+        <v-stepper
+          class="pa-4"
+          v-model="stp"
+          vertical>
+          <v-stepper-step
+            class="text-uppercase"
+            :complete="stp > 1"
+            step="1">
+            {{ $t('processes.wroks.form.label.description') }}
```
Comment (mkrout) [inline processes-webapp/src/main/webapp/vue-app/processes/components/AddWorkFlowDrawer.vue]: works

File: processes-webapp/src/main/webapp/vue-app/processes/components/Processes.vue [hunk 3]
```diff
@@ -42,8 +49,11 @@ export default {
   data () {
     return {
       tab: 0,
-      requestTypes: [],
-      requests: [],
+      alert: false,
+      type: '',
+      message: '',
+      workflows: [],
+      wroks: [],
```
Comment (mkrout) [inline processes-webapp/src/main/webapp/vue-app/processes/components/Processes.vue]: work

File: processes-webapp/src/main/webapp/vue-app/processes/components/Processes.vue [hunk 4]
```diff
@@ -83,12 +99,18 @@ export default {
       this.limit = this.limit || this.pageSize;
       this.loading = true;
       return this.$processesService
-        .getRequets(filter, 0, 0, expand)
-        .then(requests => {
-          this.requests = requests || [];
+        .getWorks(filter, 0, 0, expand)
+        .then(works => {
+          this.wroks = works || [];
```
Comment (mkrout) [inline processes-webapp/src/main/webapp/vue-app/processes/components/Processes.vue]: works

File: processes-webapp/src/main/webapp/vue-app/processes/components/Work.vue [hunk 5]
```diff
@@ -80,14 +80,14 @@
...
-    request: {
+    work: {
       type: Object,
       default: null,
     },
   },
   computed: {
     statusColor() {
-      return this.request.status === 'accepted' || this.request.status === 'Done' ? 'green' : this.request.status === 'pending' || this.request.status === 'WaitingOn' ? 'orange' : this.request.status === 'refused' ? 'red' : 'grey';
+      return this.works.status === 'accepted' || this.works.status === 'Done' ? 'green' : this.works.status === 'pending' || this.works.status === 'WaitingOn' ? 'orange' : this.works.status === 'refused' ? 'red' : 'grey';
```
Comment (mkrout) [inline processes-webapp/src/main/webapp/vue-app/processes/components/Work.vue]: Should be work not works

**Review by mkrout — CHANGES_REQUESTED** [review]

**Review by mkrout — APPROVED** [review]
LGTM

**Pull request merged**